task.sh

```diff
@@ -1,43 +1,61 @@
 #!/bin/bash
 
-# создаём каталог task с вложенными директориями
-# task
-#   dir1
-#   dir2
-#   dir3
-#       dir4
+# Удаляем старую директорию task, если существует
+rm -rf task
 
-# изменяем текущую директорию на task
+# Создаём структуру каталогов
+mkdir -p task/dir3/dir4
```

Comment:
Можно в одну команду
```suggestion
mkdir -p task/dir1 task/dir2 task/dir3/dir4
```

Reply:
+